pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -154,6 +163,10 @@ func (s *Service) HandleInbound(msg *service.DIDCommMsg) error {
 	if !current.CanTransitionTo(next) {
 		return fmt.Errorf("invalid state transition: %s -> %s", current.Name(), next.Name())
 	}
+	// trigger message events
```

Comment:
How is this relevant to your change?

Reply:
This was picked wrongly in rebase. Removed